src/libraries/Directory.Build.props

```diff
@@ -329,4 +329,9 @@
     <CLSCompliant Condition="'$(CLSCompliant)' == ''">true</CLSCompliant>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <ILLinkTrimAssemblyArtifactsRootDir>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)</ILLinkTrimAssemblyArtifactsRootDir>
```

Comment:
Any particular reason why you moved these over here? It probably makes more sense to keep these two property definitions on illink.targets where the rest of the code plus the things that consume it live.

Reply:
Turns out we can move the properties to `illink.targets` since it is imported in `llink-sharedframework.targets`.